Templates/TodoList-NodeJS/function.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "table",
```

Comment:
I don't think we should have a template that shows people how to do the wrong thing - in this case perform unecessary/inefficient operations. The fact is that a Function isn't suited to a general REST endpoint (either Node or C#).